gems/unicode.md

```diff
@@ -1,80 +1,65 @@
-# Unicode in D
+# Dにおけるユニコード
 
-Unicode is a global standard for encoding and representing text
-in computers. D fully supports Unicode in both the language and
-the standard library.
+ユニコードはエンコーディングの世界標準で、コンピュータ上でテキストを表現します。
```

Comment:
@kotet @kubo39 the third link you gave calls Unicode a character encoding set. What am I missing?